packages/components/src/GuidedTour/GuidedTour.component.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Tour from 'reactour';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { translate } from 'react-i18next';
+
+import Action from '../Actions/Action';
+import I18N_DOMAIN_COMPONENTS from '../constants';
+import getDefaultT from '../translate';
+
+import theme from './GuidedTour.scss';
+
+function getTooltipContent({ header, body }) {
+	return reactourCallbacks => (
+		<React.Fragment>
+			{header && <h2 className={classNames(theme.header, 'tc-guided-tour__header')}>{header}</h2>}
+			<p className={classNames(theme.body, 'tc-guided-tour__body')}>
+				{typeof body === 'function' ? body(reactourCallbacks) : body}
+			</p>
+		</React.Fragment>
+	);
+}
+
+function formatSteps(steps) {
+	return steps.map(step => {
+		const { header, body } = step.content;
+		const formattedStep = { ...step };
+		formattedStep.content = getTooltipContent({ header, body });
+		return formattedStep;
+	});
+}
+
+function getLastStepNextButton(t) {
+	return (
+		<Action bsStyle={'info'} label={t('GUIDEDTOUR_LAST_STEP', { defaultValue: 'Let me try' })} />
+	);
+}
+
+function GuidedTour({ className, disableAllInteractions, steps, t, tReady, ...rest }) {
+	if (!tReady || !steps || !steps.length) {
+		return null;
+	}
+
+	return (
+		<Tour
+			className={classNames(
+				theme['tc-guided-tour'],
+				'tc-guided-tour',
+				{ [theme['no-interaction']]: !!disableAllInteractions },
+				{ 'tc-guided-tour--no-interaction': !!disableAllInteractions },
+				className,
+			)}
+			closeWithMask={false}
+			disableDotsNavigation
+			disableInteraction
+			highlightedMaskClassName={classNames(theme.highlighted, 'tc-guided-tour__highlighted-mask')}
```

Comment:
Just in case, do you prefer to remove it?

Reply:
This is internal code, so if it doesn't exist, yes I prefer to remove it, instead of passing undefined.
It doesn't depend on any outside code, let's add it if one day we want to add style in the component.